tests/fuzz/fuzz-initial_channel.c

```diff
@@ -0,0 +1,79 @@
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+#include <unistd.h>
+#include <tests/fuzz/libfuzz.h>
+
+#include <bitcoin/pubkey.h>
+#include <bitcoin/chainparams.h>
+#include <bitcoin/script.h>
+#include <bitcoin/tx.h>
+#include <ccan/tal/tal.h>
+#include <common/amount.h>
+#include <common/fee_states.h>
+#include <common/initial_channel.h>
+#include <common/setup.h>
+#include <common/status.h>
+#include <common/utils.h>
+#include <stdio.h>
+#include <wire/wire.h>
+
+void init(int *argc, char ***argv)
+{
+	common_setup("fuzzer");
```

Comment:
if you had a teardown function you could call the other `common_teardown` (or whatever the name of it is) there